napari/components/_viewer/model.py

```diff
@@ -293,6 +293,230 @@ def add_layer(self, layer):
 
         self.layers.unselect_all(ignore=layer)
 
+    def add_image(self, image, *args, **kwargs):
+        """Add an image layer to the layers list.
+
+        Parameters
+        ----------
+        image : np.ndarray
+            Image data.
+        meta : dict, optional
+            Image metadata.
+        multichannel : bool, optional
+            Whether the image is multichannel. Guesses if None.
+        name : str, keyword-only
+            Name of the layer.
+        clim_range : list | array | None
+            Length two list or array with the default color limit range for the
+            image. If not passed will be calculated as the min and max of the
+            image. Passing a value prevents this calculation which can be
+            useful when working with very large datasets that are dynamically
+            loaded.
+        **kwargs : dict
+            Parameters that will be translated to metadata.
+
+        Returns
+        -------
+        layer : :class:`napari.layers.Image`
+            The newly-created image layer.
+        """
+        layer = layers.Image(image, *args, **kwargs)
+        self.add_layer(layer)
+        return layer
+
+    def add_pyramid(self, pyramid, *args, **kwargs):
+        """Add image pyramid layer.
```

Comment:
this looks different from the other summaries

Reply:
what do you mean here?

Reply:
compare them and you'll see – they all follow a pattern/template that this does not

Reply:
I feel like this one is almost identical to the `Image` summary, apologies for completely missing this, but I'm not seeing it. Can you help me out

Reply:
ah I see it!

Reply:
oh wait no i don't :-(

Reply:
do you mean that that line should be
`Add a pyramid layer to the layers list`?
If so I'm happy to fix!

Reply:
done